<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckBoxSettingsForm</class>
 <widget class="QWidget" name="CheckBoxSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="disabledLabel">
      <property name="text">
       <string>Disabled</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="disabledCheckBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelPositionLabel">
      <property name="text">
       <string>Label position</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="labelPositionComboBox">
      <item>
       <property name="text">
        <string>Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Right</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelTextLabel">
      <property name="text">
       <string>Label text</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="labelTextLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="checkedLabel">
      <property name="text">
       <string>Checked</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="checkedCheckBox"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="useThemeColorsLabel">
      <property name="text">
       <string>Use theme colors</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="useThemeColorsCheckBox"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="checkedColorLabel">
      <property name="text">
       <string>Checked color</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="checkedColorLineEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="checkedColorToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="uncheckedColorLabel">
      <property name="text">
       <string>Unchecked color</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="uncheckedColorLineEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="uncheckedColorToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="textColorLabel">
      <property name="text">
       <string>Text color</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="disabledColorLabel">
      <property name="text">
       <string>Disabled color</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="textColorLineEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="textColorToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="disabledColorLineEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="disabledColorToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
